response_operations_ui/templates/partials/header.html

```diff
@@ -1,33 +1,60 @@
 <div class="skip">
   <a class="skip__link" href="#main">Skip to content</a>
 </div>
-
-<header class="page__header">
-
-  {% include 'partials/header-logo.html' %}
-  {% include 'partials/banner.html' %}
-
-  <div class="bar bar__sitenavigation" role="banner">
-    <div class="bar__inner container">
-      <div class="sitenavigation">
-        <ul class="sitenavigation__list">
-          <li class="sitenavigation__listitem {% if request.path == '/' %} sitenavigation__listitem--active{% endif %}">
-            <a href="/">Home</a>
-          </li>
-          <li class="sitenavigation__listitem{% if request.path.startswith('/surveys') %} sitenavigation__listitem--active{% endif %}">
-            <a href="/surveys/">Surveys</a>
-          </li>
-          <li class="sitenavigation__listitem{% if request.path.startswith('/reporting-units') %} sitenavigation__listitem--active{% endif %}">
-            <a href="/reporting-units/">Reporting units</a>
-          </li>
-          <li class="sitenavigation__listitem{% if request.path.startswith('/respondents') %} sitenavigation__listitem--active{% endif %}">
-            <a href="/respondents/">Respondents</a>
-          </li>
-          <li class="sitenavigation__listitem{% if request.path.startswith('/messages') %} sitenavigation__listitem--active{% endif %}">
-            <a href="/messages/" id="messages-link">Messages</a>
-          </li>
-        </ul>
+<header class="header header--internal header--thin u-mb-no">
+  <div class="header__top" role="banner">
+    <div class="container container--wide">
+      <div class="grid grid--gutterless">
+        <div class="grid__col col-6@s">
+          <div class="logo">
+            {% set logo_print_cdn = cdn_url_prefix~"/img/ons-logo-black.svg" %}
+            <img src="{{ logo_print_cdn }}" alt="Office for National Statistics logo" class="logo__img header__logo--print">
+            <a href="/">
+              {% set logo_cdn = cdn_url_prefix~"/img/ons-logo-white.svg" %}
+              <img src="{{ logo_cdn }}" alt="Office for National Statistics logo" class="logo__img header__logo">
```

Comment:
alt text should be "Office for National Statistics" (no " logo") if we want to be consistent with the homepage